test/javascripts/acceptance/chat-test.js

```diff
@@ -1808,6 +1808,21 @@ acceptance("Discourse Chat - Direct Message Creator", function (needs) {
       return helper.response([]);
     });
   });
+
+  test("Create a direct message", async function (assert) {
+    await visit("/latest");
+    await click(".header-dropdown-toggle.open-chat");
+    await click(".topic-chat-drawer-header__return-to-channels-btn");
+    assert.ok(
+      !exists(".new-dm.btn-floating"),
+      "mobile floating button should not exist on desktop"
```

Comment:
are we actually on mobile view here?

Reply:
I was intending this to be on desktop view, checking to ensure we don't see the floating button which should only appear on mobile.

Reply:
Sorry I mixed with another file, ignore this one